src/components/Header/index.tsx

```diff
@@ -450,7 +453,9 @@ function Header() {
               ↗
             </Text>
           </HeaderMobileLink>
-
+          <HeaderMobileLink id="pools-nav-link" to="/pools">
+            {t('Pools V2')}
+          </HeaderMobileLink>
```

Comment:
Html ids should be unique, something like this would work.

```suggestion
          <HeaderMobileLink id="pools-nav-link-mobile" to="/pools">
            {t('Pools V2')}
          </HeaderMobileLink>
```